pkg/dashboard/ui/src/app/app.run.js

```diff
@@ -89,7 +89,7 @@ limitations under the License.
                 backendOptions: [
                     {
                         expirationTime: ConfigService.i18nextExpirationTime,
-                        defaultVersion: 'v0.7'
+                        defaultVersion: 'v0.71'
```

Comment:
Are you sure it's "71" and not "7.1"?

Reply:
@TomerShor version number is just an indicator that we need to reload resource files